sources/packages/api/src/services/notification/gc-notify.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@nestjs/common";
+import { GCNotify } from "../../types";
+import { ConfigService } from "..";
+import axios from "axios";
+import { GCNotifyResult, RequestPayload } from "./gc-notify.model";
+
+@Injectable()
+export class GCNotifyService {
+  constructor(private readonly configService: ConfigService) {}
+
+  get config(): GCNotify {
+    return this.configService.getConfig().gcNotify;
```

Comment:
This can be executed once inside the constructor and kept in a private readonly GCNotify variable.